Lab 7.ipynb

```diff
@@ -0,0 +1,854 @@
+      "text/plain": [
+       "<Figure size 1080x6840 with 14 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n_higgs = 100 / 100000\n",
+    "n_qcd = 20000 / 100000\n",
+    "\n",
+    "fig, ax = plt.subplots(14, 1, figsize = (15, 95))\n",
+    "\n",
+    "j = 0\n",
+    "for key in qcd:\n",
+    "    n_higgs = 100 / 100000\n",
+    "    n_qcd = 20000 / 100000\n",
+    "\n",
+    "    qcd_weight = np.array([n_qcd] * len(qcd[key]))\n",
+    "    higgs_weight = np.array([n_higgs] * len(higgs[key]))\n",
+    "\n",
+    "    ax[j].hist([qcd[key], higgs[key]], bins = 100, density = True, stacked = True, weights = [qcd_weight, higgs_weight])\n",
+    "#         ax[j].hist(qcd[key], bins = 100, density = True)\n",
+    "#         ax[j].hist(higgs[key], bins = 100, density = True)\n",
+    "    ax[j].set_title(key)\n",
+    "    ax[j].legend([\"QCD\", \"Higgs\"])\n",
+    "    \n",
+    "    j = j + 1\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "7ce83ec6-5df2-45da-b4a1-1f92cd78166c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "pt with equally sized bins:\n",
```

Comment:
Explain what 'equally sized bins' means // that you're checking the value in each bin as the bin size varies. Also explain what the printouts in brackets are